src/web/components/Cards/Card/components/ContentWrapper.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { css, cx } from 'emotion';
+
+const sizingStyles = css`
```

Comment:
I've been working on this some more to try and clean things up a bit. It might be worth coming back to this once those changes land.

Reply:
This is one area where a more styled components approach would shine. You could do:

```
import {Flex} from './some/where';

const CardWrapper = styled(Flex)`
    // CardWrapper now has the styles from Flex merged
        in with whatever specific styles you set here
`;
```

But in this case, sans 'styled', I guess `ContentWrapper` works by itself as a card specific wrapper because if you made it generic you'd have the problem of the side padding